news/tests.py

```diff
@@ -303,3 +327,20 @@ def test_not_published_article(self):
         self.add_permission('change_article')
         response = self.client.get(reverse('article', kwargs={'pk': self.article.pk}))
         self.assertEqual(response.status_code, 200)
+
+def create_ticketholders_for(event, n_tickets=10):
+    """Creates a list of active and inactive ticketholders for `event`"""
+    ticketholders = []
+    for i_ticket in range(n_tickets):
+        username = get_random_string(length=14)
+        email = f"{username}@example.com"
+        user = User.objects.create_user(username=username,email=email)
+        if isinstance(event, Event):
+            ticketholder = EventTicket.objects.create(user=user, event=event, active=randint(0,1))
+        elif isinstance(event, TimePlace):
+            ticketholder = EventTicket.objects.create(user=user, timeplace=event, active=randint(0,1))
+        else:
+            raise NotImplementedError()
+        ticketholders.append(ticketholder)
+    return ticketholders
```

Comment:
I have done something similar. I put the `user_and_ticket_state_tuples ` as an input parameter, so that it is more general. Nice trick with the kwargs dictionary, I had not thought of that! Thanks :)